rpcs3/Emu/Memory/vm.cpp

```diff
@@ -39,10 +41,10 @@ namespace vm
 	}
 
 	// Emulated virtual memory
-	u8* const g_base_addr = memory_reserve_4GiB(reinterpret_cast<void*>(0x2'0000'0000));
+	u8* const g_base_addr = memory_reserve_4GiB(reinterpret_cast<void*>(0x2'0000'0000), 0x2'0000'0000, 0x2'0000'0000);
```

Comment:
It doesn't seem to need alignment more than default 0x100000000, this is overengineering...

Reply:
It makes it possible to make a base ptr to a super by ORing bit 32, super ptr to base by NANDing bit 32.

Reply:
Okay, but you could just simply add or subtract 0x10000000? There isn't much difference, both are trivial operations.